coreutils/Makefile

```diff
@@ -467,3 +470,53 @@ csv-exp-201609:
 		done ; \
 	done
 
+# This target is used for running coreutils experiment with LLBMC.
```

Comment:
Maybe it would be good to add:
`# Arguments in PROGRAM_ARGS have lengths matching that of KLEE runs`